### PR TITLE
Add customized updater tutorial

### LINKED ISSUE #179: how to set custom lr updater?
Hi, 
I want to use 'StepLrUpdaterHook', but I do not want it decrease to 0.1 * lr at the step I specified. What I want is base_lr = 0.1, and then the flowing decreased lr is 0.5 * base_lr, 0.1 * base_lr, 0.05 * base_lr, 0.001 * base_lr. 
How can i do it?

Thanks in advance!

### PR DESCRIPTION
**Pull request by SuX97**
Fixes #179

### CODE REVIEW

File: docs/tutorials/new_modules.md [hunk 1]
```diff
@@ -218,3 +218,52 @@ To use it, modify the `loss_xxx` field. Since MyLoss is for regression, we can u
 ```python
 loss_bbox=dict(type='MyLoss'))
 ```
+
+## Add a new learning rate scheduler(updater)
```
Comment (innerlee) [inline docs/tutorials/new_modules.md:222]: space before `(`

File: docs/tutorials/new_modules.md [hunk 2]
```diff
@@ -218,3 +218,52 @@ To use it, modify the `loss_xxx` field. Since MyLoss is for regression, we can u
...
+The default manner of constructing a lr updater(namely, 'scheduler' by pytorch convention), is to modify the config such as:
```
Comment (innerlee) [inline docs/tutorials/new_modules.md:223]: and here

File: docs/tutorials/new_modules.md [hunk 3]
```diff
@@ -218,3 +218,52 @@ To use it, modify the `loss_xxx` field. Since MyLoss is for regression, we can u
...
+        cfg.lr_config,
+        optimizer_config,
+        cfg.checkpoint_config,
+        cfg.log_config,
+        cfg.get('momentum_config', None))
+...
+```
+So far, the supported updaters can be find in [mmcv](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/lr_updater.py), but if you want to customize a new learning rate updater, you may follow the steps below:
+
+1. First, write your own LrUpdaterHook somewhere in the codebase. The snippet followed is an example of cumtomized lr updater that uses learning rate based on a specific learning rate ratio: `lrs`, by which the learning rate decreases at each `steps`:
```
Comment (dreamerlin) [inline docs/tutorials/new_modules.md:242]: better to write in `$MMAction2/mmaction/core/lr`

**Comment by dreamerlin** [comment]
Remember to update the changelog.md

**Review by innerlee — APPROVED** [review]

**Pull request merged**